rastervision_pytorch_learner/rastervision/pytorch_learner/classification_learner.py

```diff
@@ -16,7 +15,6 @@
 from rastervision.pipeline.config import ConfigError
 
 warnings.filterwarnings('ignore')
-matplotlib.use('Agg')
```

Comment:
Yes, it automatically selects the Agg backend when imported inside the docker container. Even if it selects a GUI-capable backend like QTAgg, it shouldn't cause any problems because we're not calling `plt.show()` anywhere.